doc/document/PortingToFreeRTOSAWS.md

```diff
@@ -0,0 +1,298 @@
+# FreeRTOS AWS porting guide
+
+## Introduction
+
+The FreeRTOS Cellular Library exposes the capability of cellular modems with AT command interface. This guide provides a method to integrate FreeRTOS Cellular Library with FreeRTOS AWS. We use windows simulator and BG96 cellular module as an example. FreeRTOS Cellular Library is sub-moduled in vendor’s porting folder. To communicate with cellular modem, an implementation of comm interface is required. There are some comm interface implementations can be referenced to reduced porting effort. An example implementation of secure sockets with FreeRTOS cellular APIs is also provided in this guide. FreeRTOS AWS demo application can make use of secure sockets cellular to connect to AWS IoT Core.
+
+
+> Windows simulator and FreeRTOS AWS 202012.00 are used in this example. User may need to adapt to different platform or FreeRTOS AWS version.
+
+> This document is provided as an example. User doesn't need to follow this porting method if the have their own implementation.
+
+## Porting steps

```

Comment:
In this port, since we don't add a new Cellular network type, should we mention that cellular_setup is doing what network manager is doing?